### DISCUSSION TITLE
How often need I verify my email

### DISCUSSION THREAD

**Discussion by sigsbee** [community/community#21869] [New to GitHub]
<p>I have only one email address, and verified, but I still receive notify that my email need verify.</p>
<p>I do nothing to my email and I have verified the same email address 2 more time after I verified it.</p>
<p>What has happend?</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/sigsbee">@sigsbee</a>,</p>
<p>When mails to you bounce, GitHub will mark your mail address as undeliverable and require you to re-verify your address. This is to make sure there’s a valid mail address on file, so you can be reache for e.g. password resets or other important mails.</p>
<p>Sincerely,</p>
<p>Dennis</p>
